src/main/asciidoc/http.adoc

```diff
@@ -1083,7 +1083,7 @@ Alternatively you can just parse the `Set-Cookie` headers yourself in the respon
 ==== 30x redirection handling
 
 The client can be configured to follow HTTP redirections: when the client receives an
-`301`, `302`, `303` or `307` status code, it follows the redirection provided by the `Location` response header
+`301`, `302`, `303` or `307`, `308` status code, it follows the redirection provided by the `Location` response header
```

Comment:
I think actually this statement is incorrect (the original form), it should be:

The client can be configured to follow HTTP redirections provided by the `Location` response header when the client receives:

- a `301`, `302`, `307` or `308` status code along with a HTTP `GET` or `HEAD` method
- a `303` status code, in addition the directed request perform an HTTP `GET` method

Reply:
Thank you for your reply. I have changed it and submitted it.